liquidsoap.spec

```diff
@@ -24,8 +24,9 @@
 
 Name:     liquidsoap 
 Version:  1.3.4
-Release:  1%{?dist}
-Summary:  Liquidsoap by Savonet
+Release:  2%{?dist}
+Summary:  Audio and video streaming language
+
 License:  GPLv2
 URL:      http://liquidsoap.info/
 Source0:  https://github.com/savonet/liquidsoap/releases/download/%{version}/liquidsoap-%{version}.tar.bz2
```

Comment:
Consider to change the `Source0` URL to: `https://github.com/savonet/%{name}/releases/download/%{version}/%{name}-%{version}.tar.bz2`

Reply:
Fixed in 23e5676